### PR TITLE
Fix bug show tracking format and search input form in header

### PR DESCRIPTION
**Pull request by NguyenMinhh**
- Disabled button when logging
   - LoginForm.jsx
 - Remove tracking format
   - TrackingCheckableTable.jsx
image
   - TrackingCreationTable.jsx
image
- Fix display format when click 'xem van don' in flight detail modal
image
 - Fix layout of search input in header
   - SearchForm.jsx
   - ui/reducers/modules/search.js
   - ui/types/index.js
image
image
 - Disabled button when update home page data (func: quan ly trang chu)
   - EditHomePage.jsx
   - UploadImgComponent.jsx
 - Remove old missing tracking when found tracking in VN
   - classes/tracking/tracking.js

### CODE REVIEW

File: lib/classes/tracking/tracking.js [hunk 1]
```diff
@@ -280,5 +280,6 @@ export default class TrackingWorker {
 
   findMissingTrackingInVNHandler(flightId) {
```
Comment (hoadinh2010) [inline lib/classes/tracking/tracking.js]: Why don't we just use ` findMissingTrackingInVN`

File: lib/classes/tracking/tracking.js [hunk 2]
```diff
@@ -280,5 +280,6 @@ export default class TrackingWorker {
...
     const trackings = this.getListMissingTracking(flightId);
+    this.removeListMissingTrackingByListTracking(trackings);
```
Comment (hoadinh2010) [inline lib/classes/tracking/tracking.js]: The name of function is ` findMissingTrackingInVN`. But inside the function, you don't return any record ?

File: lib/classes/tracking/tracking.js (hunk 1 above)
Comment (NguyenMinhh) [reply lib/classes/tracking/tracking.js]: I'll improve this function name in PR https://github.com/hoadinh2010/Vietexpress/pull/219

File: lib/classes/tracking/tracking.js (hunk 2 above)
Comment (NguyenMinhh) [reply lib/classes/tracking/tracking.js]: I'll refactor code in PR https://github.com/hoadinh2010/Vietexpress/pull/219

**Pull request merged**